### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Efiamotu-1**
# Features Added
>- menu.js file for javascript open and close menu functions
>- eslint to check for javascript errors
>- css styling to manage open and close change on the navigation links
>- close icon to close the menu when clicked

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```
Comment (HAMMAS-SALEEM) [inline .eslintrc.json:25]: - [ ] You have set up the linters but not completely. You also need to setup the eslint to check javascript files which is missing in your pull request. Please add all the linters from this [site](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Pull Request :red_circle:

You have made great progress but I cannot review your pull request because you are missing general requirements. See the inline comment below :arrow_down:

**_This won't be counted as a review because pull request is invalid :red_circle:_**

File: css/style.css [hunk 2]
```diff
@@ -248,75 +296,48 @@ padding: 10px 20px;
...
     background-position: center;
     background-size: cover;
     height: 400px;
     display: grid;
     grid-template-rows: repeat(1, 1fr);
     align-items: flex-end;
+  }
+
+  .first-project {
+    background-image: url(images/clear-bg.png);
```
Comment (sumairq) [inline css/style.css]: - [ ] You need to make sure the background images are displaying correctly. Please fix the URLs in `background-image` 
In order to fix you need to add `""` around the image URLs and add `/` at the start of URLs
### Do the following to all the background image URLs in your project:
**For example:**
In your `style.css` file at `line 108`  `background-image: url(images/illustration-header-mobile.png);` should be `background-image: url("/images/illustration-header-mobile.png");`

File: css/style.css [hunk 3]
```diff
@@ -53,7 +54,54 @@
...
+    /* display: none; */
+    background: #3C3A39;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    z-index: 1;
+    padding: 30px;
+  }
```
Comment (sumairq) [inline css/style.css:72]: - [ ] I noticed that the mobile menu does not disappear after resizing to the desktop width. Kindly fix this issue.
Cheers 👍 

Habeeb's Portfolio

**Review by sumairq — CHANGES_REQUESTED** [review]
### Hi @Efiamotu-1 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Interactive elements are working correctly
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi Team 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥🔥


## Highlights ✨✨
- No Linters error ✅✅
- Clean code ✅✅
- Great working on the Functionality ✅✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**